devnet/vitosha-3/currencies.json

```diff
@@ -212,75 +232,163 @@
                     "network": "JUNO",
                     "ch": "channel-190"
                 }
+            },
+            {
+                "a": {
+                    "network": "NOLUS",
+                    "ch": "channel-116"
+                },
+                "b": {
+                    "network": "NEUTRON",
+                    "ch": "channel-209"
+                }
+            },
+            {
+                "a": {
+                    "network": "NEUTRON",
+                    "ch": "channel-8"
+                },
+                "b": {
+                    "network": "AXELAR",
+                    "ch": "channel-???"
+                }
+            },
+            {
+                "a": {
+                    "network": "NEUTRON",
+                    "ch": "channel-1"
+                },
+                "b": {
+                    "network": "COSMOS_HUB",
+                    "ch": "channel-???"
+                }
             }
         ]
     },
-    "lease": {
-        "Lpn": ["USDC"],
-        "Lease": {
-            "ATOM": {
-                "swap_routes": [
-                    [
-                        {
-                            "pool_id": "12",
-                            "pool_token": "OSMO"
-                        },
-                        {
-                            "pool_id": "5",
-                            "pool_token": "USDC"
-                        }
+    "protocols": {
+        "osmosis_axlusdc": {
+            "DexNetwork": "OSMOSIS",
+            "Lpn": {
+                "dex_currency": "USDC_AXELAR"
```

Comment:
If we only support one LPN currency per protocol, this should be rather `"Lpn": "USDC_AXELAR"`.

Reply:
right, we support only one LPN per protocol. The current definition does not allow multiple currencies. Here the intent is to stress the fact that Lpn is a currency like the other for the protocol, and its structure/definition is the same moludo the optionality of `swap_routes`.